xprof/xprof.rb.in

```diff
@@ -661,19 +677,18 @@ def trace_and_on_node_processing(usr_argv)
     # Only local master spawn LTTNG daemon and start session
     lm_setup_lttng(backends) if mpi_local_master?
     syncd.local_barrier('waiting_for_lttng_setup')
+
     # Launch User Command
-    launch_usr_bin(h, usr_argv)
+    begin
+      launch_usr_bin(h, usr_argv)
+    rescue Errno::ENOENT
+      teardown_lttng(syncd)
+      exit(1)
```

Comment:
This means that `Sync_daemon` `ensure` is now called, but that ok.  The spawned process is a child and will be killed, too.  We can always do the `exit(1)` inside the `Sync_daemn open` and `raise` in this code, if cleaner